htdocs/support/see_request.bml

```diff
@@ -289,7 +289,7 @@ body<=
             my $media_usage = DW::Media->get_usage_for_user( $u );
             my $media_quota = DW::Media->get_quota_for_user( $u );
             my $megabytes  = sprintf( "%0.3f MB", $media_usage / 1024 / 1024 );
-            my $percentage = sprintf( "%0.1f%%", $media_usage / $media_quota * 100 );
+            my $percentage = sprintf( "%0.1f%%", $media_usage / ( $media_quota || 1 ) * 100 );
```

Comment:
In this case since we're just printing it, I'd rather see something like 'if quota == 0, then percentage == `no quota`' or something?